modules/ossm-installing-multi-primary-multi-network-kiali.adoc

```diff
@@ -0,0 +1,216 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-multi-primary-multi-network-kiali_{context}"]
+= Installing a multi-cluster mesh by using Kiali
```

Comment:
nit: we're not installing the multi-cluster mesh here. We're installing _Kiali_ into a multi-cluster deployment.

Reply:
done